oaknational/Oak-Web-Application
# [CUR-614] Extract unit pages into code and fixed unit options

## Description

- Extracted unit pages from template document into code to easy development
- Added correct "unit option" styling.

## Issue(s)

`CUR-614`

## How to test

1. Go to {owa_deployment_url}/teachers/curriculum/docx-poc/select
2. Choose a subject via the autoComplete
3. Download the docx template from https://www.notion.so/oaknationalacademy/Curriculum-Downloads-Engineering-Spike-CUR-422-5edfb7d91b1c40a292a29b1d6cc4366a?pvs=4#d19cf1458474443188b824a8676d2804
   - Please note use the PR 2459 version
4. Download the rendered document
5. Check
   - the unit table and unit details are interleaved and contain the correct data
   - the unit options are styled correctly 


## Screenshots

<img width="527" alt="Screenshot 2024-06-03 at 09 57 33" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/41ea7eb8-e424-4147-8c4c-3ab1b6a98e34">


## Checklist
